django_guid/utils.py

```diff
@@ -91,3 +92,22 @@ def validate_guid(original_guid: str) -> bool:
         return bool(uuid.UUID(original_guid, version=4).hex)
     except ValueError:
         return False
+
+
+def is_url_in_ignored_list(request: Union['HttpRequest', 'HttpResponse']) -> bool:
+    """
+    Support for Regex added
+    Checks if the current URL is defined in the `IGNORE_URLS` setting.
+
+    :return: Boolean
+    """
+    endpoint = request.path.strip('/')
+
+    IGNORE_URLS = []
+    for url in settings.ignore_urls:
+        url_regex = url.replace('*', r'[\s\S]*')  # noqa
+        url_regex = '^' + url_regex + '$'
```

Comment:
If we make this a new setting, e.g. `ignore_url_regex`, we can also skip this.

Reply:
@JonasKs Thanks for the suggestions. I have moved it to a new settings. But facing errors in CI pipeline.

Reply:
I’m on vacation, but I’ll check when I’m back. 😊